### PR TITLE
TeamCollection Class/Test files

### PR DESCRIPTION
**Pull request by aperezsantos**
Created team collection class/test files
Add test to confirm team objects are created

### CODE REVIEW

File: lib/team_collection.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+require 'CSV'
+
+class TeamCollection
+  attr_reader :teams
+
+  def initialize(team_file_path)
+    @teams = create_teams(team_file_path)
+  end
+
+  def create_teams(team_file_path)
```
Comment (mikez321) [inline lib/team_collection.rb:11]: This seems like a good place for inheritance or a module in the future.  But works for now!

File: test/team_collection_test.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    @team_file_path = './data/teams.csv'
+    @team_collection = TeamCollection.new(@team_file_path)
+  end
+
+  def test_it_exists
+
+    assert_instance_of TeamCollection, @team_collection
+  end
+
+  def test_it_creates_team_objects
```
Comment (mikez321) [inline test/team_collection_test.rb:18]: Nice test, checking for instances of what objects we're expecting.

**Review by mikez321 — APPROVED** [review]

File: lib/team_collection.rb (hunk 1 above)
Comment (colintalex) [reply lib/team_collection.rb:11]: I'll second that motion!

**Review by colintalex — CHANGES_REQUESTED** [review]
Can we go through each of the files to lower-case the CSV require's? Mike mention this was throwing us errors

**Review by colintalex — CHANGES_REQUESTED** [review]
Still need to lowercase CSV requires

**Review by colintalex — APPROVED** [review]
Looks like we got it done, all lowercased CSV for require's.

**Pull request merged**